### DISCUSSION TITLE
Add Midi Label for active General Piston

### DISCUSSION THREAD

**Discussion by mutti-wilson** [GrandOrgue/grandorgue#899] [Developers]
I would like to request a label be added as a MIDI object that can be mapped to an LCD display for general position. There exists one for the combination setter and that works great but not one for the Generals. There is a label for the General Bank but not one for the selected General. I would not want to get rid of the bank label to add the General position but instead add to it.

**Comment by larspalo** [comment]
You do know that you can configure any general to send MIDI messages directly, right? Just right click on the general and configure the Send.

In my opinion it makes very little sense to display the currently active general on a label since it's visibly selected in the GUI already. And for the MIDI part, that's already possible to do even though you have to manually configure the exact message for each general you want to send from.

**Reply by mutti-wilson** [reply]
I never looked at the bottom of the send even list. I will try this out. I am using Midi Stops with a touch screen interface I've designed. It would be nice for our organists to be able to see the general they have selected at a glance since the pistons are not lighted.
![Layout](https://user-images.githubusercontent.com/92544601/145467778-a2445ffc-a426-47d2-8881-580572ab5645.png)
